packages/strapi-plugin-i18n/admin/src/containers/SettingsPage/LocaleSettingsPage.js

```diff
@@ -0,0 +1,134 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { useIntl } from 'react-intl';
+import { BaselineAlignment, ModalConfirm, EmptyState, ListButton } from 'strapi-helper-plugin';
+import { Header, List } from '@buffetjs/custom';
+import { Button, Text } from '@buffetjs/core';
+import { Plus } from '@buffetjs/icons';
+import ModalEdit from '../../components/ModalEdit';
+import { LocaleRow } from '../../components';
+import { useLocales } from '../../hooks';
+import { getTrad } from '../../utils';
+import useDeleteLocale from '../../hooks/useDeleteLocale';
+import useEditLocale from '../../hooks/useEditLocale';
+
+const LocaleSettingsPage = ({ canCreate, canDelete, canUpdate }) => {
+  const {
+    isDeleting,
+    isDeleteModalOpen,
+    deleteLocale,
+    showDeleteModal,
+    hideDeleteModal,
+  } = useDeleteLocale();
+
+  const { isEditing, isEditModalOpen, editLocale, showEditModal, hideEditModal } = useEditLocale();
```

Comment:
ℹ️ If LOCs start to become a problem, we can eventually split this hook in two: one for dealing with the modal, and one for the edit code. Then, we could only deal with the modal one in this file and move the one about the edition directly inside the `ModalEdit`. Same applies for the delete one

Reply:
I'll let you decide what you think is best.